app/components/schedule/Home.vue

```diff
@@ -44,6 +44,7 @@ export default {
       showsByDate: [],
       dayNames: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'],
       interval: null,
```

Comment:
Can we rename this to something else that tells us that this is about the clock 1 minute intervals? clockInterval? clockMinute? I trust you w/ naming it anything but after introducing `recheckInterval` this original `interval` confused me a bit

Reply:
warum nicht...